CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Surrogate` models now operate on dataframes in experimental representation instead of
   tensors in computational representation
 - `Surrogate.posterior` models now returns a `Posterior` object
+- `param_bounds_comp` of `SearchSpace`, `SubspaceDiscrete` and `SubspaceContinuous` has
```

Comment:
I think there are some more changes that should be reflected here

Reply:
Yes, PR was only 30% done so far to collect your input on the general direction. Will add more details in the end and keep this thread open as a reminder

Reply:
Have added more items